app/api/packets/helpers.ts

```diff
@@ -261,9 +260,8 @@ export async function getPackets() {
   return response;
 }
 
-export async function getPacket(txHash: string) {
+export async function getPacket(txHash: string): Promise<[] | [any]> {
```

Comment:
Why does return type has to use `any` ?

Reply:
It considers an empty array and an array of type empty two different types

Reply:
Why can't you use `Packet[]`?